### ISSUE TITLE
Implementation of equation (7) in the paper.

### ISSUE THREAD

**Issue by kai-wen-yang** [ziquanliu/NeurIPS2022_UOT_fine_tuning#1]
In equation (7) of the paper, you use pesudo labeling of two augmentations for unlabeled pre-training. I want to ask where is the implementation of this part in the code? In https://github.com/ziquanliu/NeurIPS2022_UOT_fine_tuning/blob/3519239c40dfbab692a011df6f7bc037ca75a30c/Self_Supervised_PT/fine_tune_top1.py#L451, it seems that you directly use the real label.

**Comment by ziquanliu** [comment]
Thanks for your careful reading.

That directory is the code for Data Selection w/ Labels in Table 1b. 

I just uploaded the code for Data Selection w/o Labels in [Self_Supervised_PT_FixMatch](https://github.com/ziquanliu/NeurIPS2022_UOT_fine_tuning/tree/main/Self_Supervised_PT_FixMatch).

**Comment by kai-wen-yang** [comment]
Thanks for your reply.
